src/clr-core/common/utils/enums.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export enum KeyCodes {
+  LeftArrow = 'ArrowLeft',
```

Comment:
![](https://media.giphy.com/media/5mYwgGvIR2GN2g0ZZj/giphy.gif)

It seems that all other `KeyCodes` follow the pattern of `Arrow<Direction>` and only the `LeftArrow` have this reverse pattern `<Direction>Arrow` - is it not better to have the same pattern for all directions?

I did not find a place where we are using yet, so this will be a local change only.

Reply:
Haha good find, that's what I get for to much copy pasting! 🤣 Will fix up. I have a couple other fixes as well. I'll wait on force pushing anything else.

Reply:
I think that we reach perfection for this file. :)